.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
This is because I started working with Pycharm and move to Visual Studio Code because it has a Nextflow plugin. The .idea was getting in my way which is why it made it to the `.gitignore`.